docs/release-notes/highlights-1.7.0.asciidoc

```diff
@@ -0,0 +1,34 @@
+[[release-highlights-1.7.0]]
+== 1.7.0 release highlights
+
+[float]
+[id="{p}-170-new-and-notable"]
+=== New and notable
+
+New and notable changes in version 1.7.0 of {n}. See <<release-notes-1.7.0>> for the full list of changes.
+
+[float]
+[id="{p}-170-splitted-crds"]
+==== v1 versions of CustomResourceDefinitions (CRD) and ValidatingWebhookConfiguration are available
+
+Starting with this release, the `CustomResourceDefinitions` (CRD) and the `ValidatingWebhookConfiguration` resources are available in version `v1`. The resources definitions and the operator are now provided in two separate files, the `all-in-one.yaml` file is no longer available. Check the link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-deploy-eck.html[installation guide] or the link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-upgrading-eck.html#k8s-beta-to-ga-upgrade[upgrade notes] for more information.
+
+[float]
+[id="{p}-170-stack-monitoring"]
+==== Stack Monitoring
+
+In this release, the Elasticsearch and Kibana resources have been enhanced to let you specify a reference to a monitoring cluster. When specified, sidecars containers are automatically setup by ECK to ship logs and metrics to the referenced Elasticsearch cluster.
+
+*Add a link to documentation*
+
+[float]
+[id="{p}-170-autoscaling"]
+==== Autoscaling
+
+The `/scale` subresource is now enabled for Kibana, Enterprise Search, Elastic Maps Server and APM Server. The number of replicas for these resources can now be managed automatically by the link:https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/[Horizontal Pod Autoscaler] (HPA).
```

Comment:
"now" is repeated in both sentences.  I don't have a good suggestion though :-(

Reply:
> "now" is repeated in both sentences. I don't have a good suggestion though :-(

Agreed 😕 , @kilfoyle do you have a suggestion to avoid the repetition here ? Thanks !

Reply:
Sure! I would just remove the second `now`, and I'd also add an "Oxford comma" after `Elastic Maps Server` just for consistency with other docs:

`The `/scale` subresource is now enabled for Kibana, Enterprise Search, Elastic Maps Server, and APM Server. The number of replicas for these resources can be managed automatically by the link:https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/[Horizontal Pod Autoscaler] (HPA).`

I'd also make the second sentence active, but only if you think it sounds better:
`The `/scale` subresource is now enabled for Kibana, Enterprise Search, Elastic Maps Server, and APM Server. You can set these resources to be managed automatically by the link:https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/[Horizontal Pod Autoscaler] (HPA).`